### PR TITLE
4.8 RNs: OSUS GA blurb

### PR DESCRIPTION
**Pull request by jeana-redhat**
h/t @bobfuru for the wording :pray: 

Preview: https://deploy-preview-34120--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-osus

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@LalatenduMohanty or @jottofar - could one of you please check this summary of OSUS for the 4.8 Release Notes?

**Comment by bobfuru** [comment]
LGTM. Thanks for handling this, @jeana-redhat!

**Comment by jeana-redhat** [comment]
added Lala and Jack as reviewers - just need an ack from one to merge

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -142,6 +142,13 @@ the bare metal pod definition. See link:https://bugzilla.redhat.com/show_bug.cgi
 
 * Disabling RAID for all drivers. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1969487[BZ#1969487] for more information.
 
+[id="ocp-4-8-osus"]
+==== Over-the-air updates with the OpenShift Update Service
+
+The OpenShift Update Service (OSUS) provides over-the-air updates to {product-title}, including Red Hat Enterprise Linux CoreOS (RHCOS), as a hosted service located behind public APIs. The OpenShift Update Service is composed of an Operator and one or more application instances and is now generally available in {product-title} 4.6 and higher.
```
Comment (jottofar) [inline release_notes/ocp-4-8-release-notes.adoc:148]: Changed to:

The OpenShift Update Service (OSUS) provides over-the-air updates to {product-title}, including Red Hat Enterprise Linux CoreOS (RHCOS). It was previously only accessible as a Red Hat hosted service located behind public APIs but is now available to be installed locally. The OpenShift Update Service is composed of an Operator and one or more application instances and is now generally available in {product-title} 4.6 and higher.

/cc @LalatenduMohanty

Comment (jeana-redhat) [reply release_notes/ocp-4-8-release-notes.adoc]: Thanks, Jack! Made a bit of a tweak but don't think it changed the meaning - lmk if it's still alright.

**Comment by jeana-redhat** [comment]
GitHub ate this PR. Re-submitted as #34217 :facepalm:

**Pull request closed without merging**